changelog/README.md

```diff
@@ -8,6 +8,9 @@ For more details, see [Environments](../guidelines/environments.md).
 
 # Changelog
 
+## 12th August 2022
+* Added a `ActivityState` and `MergeTargetId` into the [Customer](../operations/customers.md#customer).
+
```

Comment:
Suggest better wording:
* Added `ActivityState` and `MergeTargetId` to the [Customer](../operations/customers.md#customer) object.